pkg/apiserver/rest/webservice/addon.go

```diff
@@ -182,7 +184,13 @@ func (s *addonWebService) enableAddon(req *restful.Request, res *restful.Respons
 
 func (s *addonWebService) disableAddon(req *restful.Request, res *restful.Response) {
 	name := req.PathParameter("name")
-	err := s.handler.DisableAddon(req.Request.Context(), name)
+	forceParam := req.QueryParameter("force")
```

Comment:
Please add a QueryParameter about "force" in route definition.

Reply:
Cannot get the point. Can you  submit an issue for this?